Analysis/config/2018/sources.cfg

```diff
@@ -26,10 +26,10 @@ jet_ordering: DeepFlavour
 #syncDataIds: sync_DY_nlo:2j/mh/OS_Isolated/Central/DY_nlo_SF_.*
 #syncDataIds: sync_DY_nlo:2j/NoCuts/OS_Isolated/Central/DY_nlo_SF_.*
 trigger_path: h-tautau/Production/data/triggers_2018.cfg
-trigger: eTau HLT_Ele32_WPTight_Gsf_v HLT_Ele35_WPTight_Gsf_v HLT_Ele24_eta2p1_WPTight_Gsf_LooseChargedIsoPFTau30_eta2p1_CrossL1_v HLT_Ele24_eta2p1_WPTight_Gsf_LooseChargedIsoPFTauHPS30_eta2p1_CrossL1_v
-trigger: muTau HLT_IsoMu24_v HLT_IsoMu27_v HLT_IsoMu20_eta2p1_LooseChargedIsoPFTau27_eta2p1_CrossL1_v HLT_IsoMu20_eta2p1_LooseChargedIsoPFTauHPS27_eta2p1_CrossL1_v
-trigger: tauTau HLT_DoubleTightChargedIsoPFTau35_Trk1_TightID_eta2p1_Reg_v HLT_DoubleMediumChargedIsoPFTau40_Trk1_TightID_eta2p1_Reg_v HLT_DoubleTightChargedIsoPFTau40_Trk1_eta2p1_Reg_v HLT_DoubleMediumChargedIsoPFTauHPS35_Trk1_eta2p1_Reg_v
+trigger: eTau HLT_Ele32_WPTight_Gsf_v HLT_Ele35_WPTight_Gsf_v HLT_Ele24_eta2p1_WPTight_Gsf_LooseChargedIsoPFTau30_eta2p1_CrossL1_v
```

Comment:
why you removed HPS triggers for 2018? The channel list as it was in the previous comment is correct. Please, undo the modifications.